cmd/operator-ui/templates/hubs/_form.html

```diff
@@ -18,6 +18,7 @@
       <%= f.TextAreaTag("Spec.Certificate", {value: hub.Spec.Certificate, label: "Certificate"}) %>
       <%= f.TextAreaTag("Spec.CertificateKey", {value: hub.Spec.CertificateKey, label: "Certificate Key"}) %>
     </div>
+    <%= f.TextAreaTag("Spec.ProxyCertificate", {value: hub.Spec.ProxyCertificate, label: "Proxy Root Certificate"}) %>
```

Comment:
@jdartigalongue 
- Is Proxy certificate a text field? 
- For each hub instance, customer need to provide the proxy certificate? 
- Whether the Certificate logic is similar to Proxy certificate?

Reply:
@msenmurugan The Hub custom resource ProxyCertificate field contains the certificate in PEM format (same as Certificate and CertificateKey). Therefore, I am using a text field in the UI. 

This field is optional so customers can leave it empty if they do use an HTTPS proxy.

Reply:
@jdartigalongue For certificates, if one hub is having a custom certificate, customer will have the ability to select that certificate for the new hub in UI and also in JSON file. Do you want to build similar kind of logic for proxy certificate?